package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "keywords": [
     "OpenShift.io",
-    "fabric8-analytics",
+    "CVE",
```

Comment:
@invincibleJai Following categories are the ones that I think are appropriate for this extension:
```
Programming Languages
Linters
Other
```

IMO, here are the five keywords that we can associate with the extension:
```
"analytics"
"dependency recommender"
"cve security vulnerability"
"license "
"node maven"
```

Also, the title description requires a change. I think "Flags security vulnerabilities, reports license anomalies and recommends dependencies for NodeJS and Maven applications." is a good way to present what this extension does. Please opine @krishnapaparaju @miteshvp @sivaavkd